### PR TITLE
Update breaking changes in spot_wrapper refactoring

### PR DESCRIPTION
**Pull request by jeremysee2**
This addresses the breaking changes from refactoring code in spot_wrapper at https://github.com/bdaiinstitute/spot_wrapper/pull/6

- Image, world_objects and graph_nav functionality have been broken into separate objects called through `spot_wrapper.{object}`
- Refactoring GraphNav navigate_to into initialization then navigate to 
a specific waypoint `spot_wrapper.spot_graph_nav.navigate_to_specific_waypoint'

### CODE REVIEW

**Comment by jeremysee2** [comment]
I've tested this on the surface level, appears to connect to the robot and authenticate fine. Would appreciate someone who uses this regularly to test it out.

File: spot_driver/spot_driver/spot_ros2.py [hunk 1]
```diff
@@ -814,9 +815,9 @@ def handle_list_graph(self, request, response):
         """ROS service handler for listing graph_nav waypoint_ids"""
         try:
             self.node.get_logger().error(f'handle_list_graph: {request}')
-            self.spot_wrapper._clear_graph()
-            self.spot_wrapper._upload_graph_and_snapshots(request.upload_filepath)
-            response.waypoint_ids = self.spot_wrapper.list_graph(request.upload_filepath)
+            self.spot_wrapper.spot_graph_nav._clear_graph()
+            self.spot_wrapper.spot_graph_nav._upload_graph_and_snapshots(request.upload_filepath)
```
Comment (amessing-bdai) [inline spot_driver/spot_driver/spot_ros2.py:819]: Is there a reason these are private functions in spot_wrapper if they are going to be used here?

File: spot_driver/spot_driver/spot_ros2.py [hunk 2]
```diff
@@ -49,6 +49,7 @@
 
 import logging
 import threading
+import traceback
```
Comment (jbarry-bdai) [inline spot_driver/spot_driver/spot_ros2.py:52]: extra import?

**Comment by heuristicus** [comment]
There are a few outstanding PRs on the choreography changes in the wrapper, probably best to get those merged in before we fix conflicts, otherwise we'll have to do it twice. Perhaps a short pause on further PRs into the wrapper until we have the modular wrapper merged. I will try and do some testing with the ROS1 version over the weekend and into next week.

**Comment by jeremysee2** [comment]
Yup, I'll look into those conflicts on the weekend.

**Comment by amessing-bdai** [comment]
The spot_wrapper PR this corresponds to was broken into pieces and several PRs were made and merged in this repo corresponding to those pieces. As such, this PR is no longer needed.

**Pull request closed without merging**